js/api/Surveys.js

```diff
@@ -28,8 +30,8 @@ export function loadAllAcceptedSurveys(callback) {
 
     const surveys = []
     const invitationsLength = invitations.length;
-    for (let i = 0; i < invitationsLength; i++) {
-      let acceptedSurvey = realm.objects('Survey').filtered(`id = "${invitations[i].surveyId}"`)[0]
+    for (var i = 0; i < invitationsLength; i++) {
```

Comment:
Another one cause by pulling `map-view` into this branch.
I'll adjust this one, but most of the linting is being done on the active branch due to their sheer number.